docs/operating-eck/troubleshooting/common-problems.asciidoc

```diff
@@ -84,3 +84,75 @@ type: Opaque
 ----
 
 Failure to do so can cause data loss.
+
+[id="{p}-{page_id}-pod-updates"]
+== Pods are not replaced after configuration update
+
+When updating an existing Elasticsearch cluster configuration, the update can fail while being be applied.  This happens when the operator is unable to apply the changes required by replacing the pods of a given the Elasticsearch cluster.
```

Comment:
```suggestion
The update of an existing Elasticsearch cluster configuration can fail because the operator is unable to apply the changes required while replacing the pods of a given Elasticsearch cluster.
```